docs/index.md

```diff
@@ -5,19 +5,55 @@ _navPath: toc.html
 _navRel: ../toc.html
 ---
 
-## Getting started
+.NET Documentation
+==================
 
-In this docset, you will find tutorials that will teach you how to get started with .NET Core. You can also find the API reference documentation. 
+Welcome to .NET! You can build any kind of app you want with .NET, from cloud to mobile to desktop to IoT to games. You can start building your next app today, for free, on Windows, OS X and Linux.
```

Comment:
I'm not opposed.  The person who's never tried .NET and is looking to get started can do so without paying a single penny.  So long as that's the context this is framed up as, it's certainly accurate.